tensorflow_datasets/image/fruits360.py

```diff
@@ -0,0 +1,125 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fruits-360: A dataset of images containing fruits.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from tensorflow.python.framework import errors
+
+import tensorflow_datasets.public_api as tfds
+from tensorflow import io as tfio
+
+_CITATION = """\
+@article{article,
+author = {Mureșan, Horea and Oltean, Mihai},
+year = {2018},
+month = {06},
+pages = {26-42},
+title = {Fruit recognition from images using deep learning},
+volume = {10},
+journal = {Acta Universitatis Sapientiae, Informatica},
+doi = {10.2478/ausi-2018-0002}
+}
+"""
+
+_COMMIT_SHA = 'b6960fc0287be349c222c3dbdd2624958c902431'
```

Comment:
what's it ? why do you need?

Reply:
I'll take a look at this later today.

Reply:
@Conchylicultor I think I'm a little confused as to how `BuilderConfig`s work. For the lm1b example, it seems like the configurations are hardcoded into the builder; are you suggesting I create a configuration for every commit, or is there a way for people to pass in their own commit?

The flip side of this is whether it'd be possible to mark the updates as incremental updates to the builder version.

Reply:
`BuilderConfig` allow to creates multiple version of a same dataset.

By default, datasets can pre-define some available configs (tfds.load(`my_dataset/config1)`) but users can also creates their own (`tfds.load(`my_dataset', builder_kwargs={'config': custom_config}`).

Here, the dataset should define a single builder config corresponding to the current last released version, but other users could create their own by creating custom `tfds.images.fruit360.Fruit360Config(...)`.

Reply:
@Conchylicultor I see. So going forward, would we keep adding `BuilderConfig`s for each new update, or simply leave it at whatever the most recent is when this PR is merged?

Reply:
Leaving to the most recent when this PR is merged is fine. We may occasionally add new versions depending on the demand. The main point is that the user could easily choose another version if they want.

For the builder config name, the best is to match the date from the [github release homepage](https://github.com/Horea94/Fruit-Images-Dataset) 
```python
BUILDER_CONFIG=[
    Fruits360Config(name='2019.08.14.0', sha=..., num_classes=...)
]

...

ds = tfds.load('fruit360/2019.08.14.0')
```

Reply:
In the doc (`DatasetInfo(description='...')`), please also add a small code snippet to explain how to use the dataset with a custom config, similarly to the code snippet from [wmt documentation page](https://www.tensorflow.org/datasets/catalog/wmt14_translate)